### PR TITLE
Added functionality to handle user login success state

### LINKED ISSUE #179: Add google authentication to home page and add state
**Is your feature request related to a problem? Please describe.**
The login button should be handled once auth succeeds. Right now it does not. 

**Describe the solution you'd like**
The google login button should simply be in the home page header. Once the JWT token is received with google response, we should handle displaying the user's name and profile picture once logged in successfully to let them know that they've logged in. 

**Additional context**
Multiple components in the login system will rely on this "hasLoggedIn" state a few features will be locked behind logging in. 
Issue discussed at team meeting.

### PR DESCRIPTION
**Pull request by itsercleung**
Fixes #179

## Proposed Changes
  - Moved login to common/Header component
  - Added handleLogin() in FrontpageContainer which sets displayProfile state
  - Added displayProfile state which has (isLoggedIn, displayName, displayImage) properties
  - Shows displayProfile in Header once google logged in

## Note
displayProfile property isLoggedIn can be used for multiple components that requires user to be logged in.

### CODE REVIEW

File: frontend/src/common/Header/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```
Comment (Sabatr) [inline frontend/src/common/Header/Login.jsx:1]: Will this file have the changes from @saddboys 's Login stuff too?

Comment (itsercleung) [reply frontend/src/common/Header/Login.jsx:1]: Did you see any additions from his merge?? I don't remember having a conflict other than from newest master

Comment (Sabatr) [reply frontend/src/common/Header/Login.jsx:1]: It seems like this PR will make a new file called `Login.jsx` in `common/Header/` but there will still be a `Login.jsx` in `pages/Login` which is what David was working on

Comment (itsercleung) [reply frontend/src/common/Header/Login.jsx:1]: Okay thats wack. I essentially, moved existing pages/Login.jsx to common/Header

Comment (itsercleung) [reply frontend/src/common/Header/Login.jsx:1]: Ill see what sup

Comment (itsercleung) [reply frontend/src/common/Header/Login.jsx:1]: Okay i've moved the function made by @saddboys!

**Pull request merged**